src/mixins/auth-api-mixin.ts

```diff
@@ -0,0 +1,42 @@
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/axios-auth'
+
+/** Mixin that provides integration with the Auth API. */
+@Component({})
+export default class AuthApiMixin extends Vue {
```

Comment:
So I thought about creating a unit test file for this (and the other 2) API mixins, but they really don't do much so we really wouldn't be testing much (other than our test code). So I didn't.

Reply:
Good call. On a regular mount i believe tests can run through code which utilize the mixins, which in a way tests them anywho right? If not always in isolation but at least in practice.

Reply:
Haha, I've actually partly changed my mind on this, in order to increase code coverage!

Ideally, in components that use these API mixins, we should mock the mixin methods. But I'm not changing that now :)